UnitsNet/CustomCode/UnitAbbreviationsCache.cs

```diff
@@ -276,9 +276,11 @@ public string[] GetUnitAbbreviations(Type unitType, int unitValue, IFormatProvid
             if(!TryGetUnitValueAbbreviationLookup(unitType, formatProvider, out var lookup))
                 return formatProvider != FallbackCulture ? GetUnitAbbreviations(unitType, unitValue, FallbackCulture) : new string[] { };
 
-            var abbreviations = lookup!.GetAbbreviationsForUnit(unitValue);
-            if(abbreviations.Count == 0)
-                return formatProvider != FallbackCulture ? GetUnitAbbreviations(unitType, unitValue, FallbackCulture) : new string[] { };
+            var abbreviations = new HashSet<string>();
+            abbreviations.UnionWith(lookup!.GetAbbreviationsForUnit(unitValue));
+
+            if(formatProvider != FallbackCulture)
+                abbreviations.UnionWith(GetUnitAbbreviations(unitType, unitValue, FallbackCulture));
```

Comment:
At first glance, I'm not sure about this union.

If you call `GetUnitAbbreviations(typeof(LengthUnit), 1, russianCulture)` then from this public method signature, isn't it unexpected to also receive English abbreviations?

Reply:
A good question :) Should it? Currently you can get English abbreviations from GetUnitAbbreviations, but only if no Russian ones exist.

The parser obviously will not fall back without it. It's either that, or the parser logic will need to include the fallback logic. I wanted to keep everything contained.

Reply:
I'm really not sure. How about adding a new parameter `bool includeFallbackCulture = false` and call it with `true` from our own usages where we need to?

Reply:
I still think we should not do a union here indiscriminately. What do you think of my proposal of a new argument?